kubetest2-ec2/pkg/deployer/up.go

```diff
@@ -124,7 +129,37 @@ func (d *deployer) Up() error {
 			}
 			return err
 		}
-		klog.Infof("starting instance id: %s", instance.instanceID)
+		if runner.controlPlaneIP == "" {
+			runner.controlPlaneIP = instance.privateIP
+		}
+		klog.Infof("started instance id: %s", instance.instanceID)
+
+		wg.Add(1)
+		go func() {
+			defer wg.Done()
+			_, err = runner.isAWSInstanceRunning(instance)
```

Comment:
```suggestion
			_, err := runner.isAWSInstanceRunning(instance)
```
Otherwise its multiple goroutines writing to the same err variable.